src/TrayIcon.cpp

```diff
@@ -129,6 +129,13 @@ TrayIcon::setUnreadCount(int count)
         previousCount = count;
     }
 #endif
+
+    QString toolTip = QLatin1String("nheko");
+    if (count > 0) {
+        toolTip.append(tr("\n%1 unread message(s)").arg(count));
```

Comment:
Sorry for the slow review, but can you please use `%n` to make the plural forms work properly in the translation: https://doc.qt.io/qt-6/i18n-source-translation.html#handle-plural-forms

Reply:
@deepbluev7 I have fixed it in my latest commit :)